docs/glossary.md

```diff
@@ -0,0 +1,97 @@
+
+## Liturgical Colors
+
+The liturgical colors are the colors used in the liturgical vestments and altar cloths for the celebration of the Mass and other liturgical services. The colors are used to signify the nature of the liturgical season or feast being celebrated.
+
+See GIRM §346.
+
+## Liturgical Day
+
+A liturgical day denotes a specific and distinct day within the Church's liturgical calendar, characterized by particular celebrations and observances. Each day is assigned a rank to indicate its significance, a specific liturgical color, and various properties that define its nature and all associated celebrations. The date of each liturgical day is determined according to the Proper of Time, the Proper of Saints, and the rules governing the precedence of other liturgical days.
+
+A liturgical day generally aligns with a day in the Gregorian calendar but has some nuances: it typically begins at First Vespers or Vigils (if observed), or otherwise at midnight, and ends at the beginning of the following liturgical day. If a liturgical day begins before midnight, it remains identified as a single liturgical day, even if it spans two civil days. A liturgical day is identified by a Gregorian date (considering the majority of the liturgical day that occurs within a given civil day).
+
+In Romcal, it is this concept of the liturgical day that is calculated within a liturgical calendar.
+
+Note: A liturgical day should not be confused with a celebration (see celebration).
+
+## Period
+
+In Romcal, in addition to the liturgical seasons, liturgical periods provide further detail and metadata about the current liturgical period. For example, the octaves of Christmas and Easter, the period before and after Epiphany, Holy Week, etc. They also denote periods traditionally observed in the Catholic Church and by religious communities (especially for Marian hymns), including the period from the Baptism of Christ to the Presentation of Jesus in the Temple, and from the Presentation of Jesus in the Temple to Holy Thursday.
+
+## Precedence
+
+The principle of precedence is used to determine which liturgical celebration should be observed when two or more celebrations fall on the same day. The principle of precedence is based on the liturgical rank of the celebrations.
+
+See GNLYC §59.
+
+## Proper of Saints (or Sanctoral)
+
+The "Proper of Saints" (or "Sanctoral") consists of the fixed feasts, celebrated on the very same date each year (no matter what the day of the week),including Christmas and all the saints' days.
+
+## Proper of Time (or Temporal)
+
+The "Proper of Time" (or "Temporal") consists of the movable feasts, most of them keyed to Easter (which falls on a different Sunday every year), including Ascension, Pentecost, and so on.
+
+Note: we avoid using the term "Temporal" to prevent potential confusion with the `Temporal` object in JavaScript, which is still in a [proposal stage](https://tc39.es/proposal-temporal/docs/).
```

Comment:
I also think we should replace `JavaScript` with `ECMAScript`.

```suggestion
> [!NOTE]
> We avoid using the term Temporal to prevent potential confusion with the `Temporal` object in JavaScript, which is still in a [proposal stage](https://tc39.es/proposal-temporal/docs/).
```

Reply:
```suggestion
> [!NOTE]
> We avoid using the term Temporal to prevent potential confusion with the `Temporal` object in ECMAScript, which is still in a [proposal stage](https://tc39.es/proposal-temporal/docs/).
```